confmap/provider/httpsprovider/testdata/RootCA.crt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
+MIICxjCCAa4CCQDUwWGijuAYTTANBgkqhkiG9w0BAQsFADAlMQswCQYDVQQGEwJV
```

Comment:
How was this generated and how long is it valid?  I think there are some `Makefile` targets for generating certs for testing.  This should either use those certs or ensure that it can be regenerated by those targets so that we don't run into issues when it expires.

Reply:
This cert is for testing uses, and generated by instructions written in my Get Started Doc. For how long it is valid, customers can setup the expiration date while generating their custom certs or buying trusted certs.

In terms of the custom certs, there is a possibility to do a Makefile! I will take a note on my TODO and Handover Doc, then work on this.

Regenerate certs is a good direction. I will take a note on my TODO and Handover Doc, then work on this.